index.html

```diff
@@ -49,5 +49,167 @@ <h1><span class="hero__title-dark">Hey there. I'm Daniel Selase</span><span clas
+                            <a href="#" class="card__tags-links">JavaScript</a>
+                            <a href="#" class="card__tags-links">html</a>
+                        </div>
+                        <button class="btn">See Project</button>
+                </div>
+                <div class="card">
+                    <div class="card__img"></div>
+                    <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+                    <div class="card__tags">
+                        <a href="#" class="card__tags-links">Ruby on rails</a>
+                        <a href="#" class="card__tags-links">css</a>
+                        <a href="#" class="card__tags-links">JavaScript</a>
+                        <a href="#" class="card__tags-links">html</a>
+                    </div>
+                    <button class="btn">See Project</button>
+            </div>
+            <div class="card">
+                <div class="card__img"></div>
+                <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+                <div class="card__tags">
+                    <a href="#" class="card__tags-links">Ruby on rails</a>
+                    <a href="#" class="card__tags-links">css</a>
+                    <a href="#" class="card__tags-links">JavaScript</a>
+                    <a href="#" class="card__tags-links">html</a>
+                </div>
+                <button class="btn">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card__img"></div>
+            <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+            <div class="card__tags">
+                <a href="#" class="card__tags-links">Ruby on rails</a>
+                <a href="#" class="card__tags-links">css</a>
+                <a href="#" class="card__tags-links">JavaScript</a>
+                <a href="#" class="card__tags-links">html</a>
+            </div>
+            <button class="btn">See Project</button>
+    </div>
+    <div class="card">
+        <div class="card__img"></div>
+        <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+        <div class="card__tags">
+            <a href="#" class="card__tags-links">Ruby on rails</a>
+            <a href="#" class="card__tags-links">css</a>
+            <a href="#" class="card__tags-links">JavaScript</a>
+            <a href="#" class="card__tags-links">html</a>
+        </div>
+        <button class="btn">See Project</button>
+</div>
+<div class="card">
+    <div class="card__img"></div>
+    <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+    <div class="card__tags">
+        <a href="#" class="card__tags-links">Ruby on rails</a>
+        <a href="#" class="card__tags-links">css</a>
+        <a href="#" class="card__tags-links">JavaScript</a>
+        <a href="#" class="card__tags-links">html</a>
+    </div>
+    <button class="btn">See Project</button>
```

Comment:
- [ ]  I think it would be better to use an unordered list to wrap using `ul` and `li` instead of div for a semantic reason.